### ISSUE TITLE
about the loss function

### ISSUE THREAD

**Issue by ZackyCaleb** [yue-zhongqi/ICON#1]
Thanks for your nice work.
As your expressions in the article indicate, the loss function consists of three items, but eqinv_loss, bce_loss, and transfer_loss are also shown in the code. Would you like to explain them?
    ```      

        loss = w_transfer * transfer_loss 

               + w_cluster * bce_loss 

               + w_eqinv * (eqinv_loss + eqinv_loss2) 

               + w_erm * cls_loss 

               + w_con * con_loss

               + w_st * st_loss

               + w_st2 * st_loss_cluster

               + w_inv * inv_loss
    ```
